helm/charts/determined/values.yaml

```diff
@@ -143,8 +142,42 @@ taskContainerDefaults:
   # (CPU notebooks, TensorBoards, zero-slot commands). If a pod spec is defined for an individual
   # task, that pod spec will replace the default one that is defined here. See
   # https://docs:determined.ai/latest/topic-guides/custom-pod-specs.html for more details.
-  # cpuPodSpec:
-  # gpuPodSpec:
+  cpuPodSpec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      labels:
+        customLabel: cw-cpu-determined-pod
+    spec:
+      containers:
+        - name: determined-cpu-container
+          resources:
+            requests:
+              memory: {{.Values.resources.cpu}}
+              cpu: {{.Values.resources.memory}}
+            limits:
+              memory: {{.Values.resources.cpu}}
+              cpu: {{.Values.resources.memory}}
+  gpu_pod_spec:
```

Comment:
this should be camelcase I think

Reply:
Will talk to @brandonrjacobs

Reply:
I meanit is called `gpuPodSpec` in the [original values file](https://github.com/coreweave/coreweave_determined/blob/master/helm/charts/determined/values.yaml#L147) so I don't think there is much to discuss, it should be `gpuPodSpec` and looking at the code https://github.com/coreweave/coreweave_determined/blob/master/helm/charts/determined/templates/master-config.yaml#L101 it is also referencing `gpuPodSpec`. What you did here does not work.

Reply:
Should be resolved.